src/views/illness/illness-start-date.njk

```diff
@@ -0,0 +1,119 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/back-link/macro.njk' import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from 'govuk/components/date-input/macro.njk' import govukDateInput %}
+
+{% block pageTitle %}
+   Appeal due to ill health: Tell us when the illness started
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form method="post">
+
+        {{
+          govukErrorSummary ({
+            titleText: 'There is a problem',
+            errorList: validationResult.errors
+          }) if validationResult and validationResult.errors.length > 0
+        }}
+
+        {% set classes_day = 'govuk-input--width-2' %}
+        {% set classes_month = 'govuk-input--width-2' %}
+        {% set classes_year = 'govuk-input--width-4' %}
+
+        {% if validationResult and validationResult.errors.length > 0 %}
+
+          {% if validationResult.getErrorForField('day') %}
+            {% set classes_day = classes_day + ' govuk-input--error' %}
+          {% endif %}
+          {% if validationResult.getErrorForField('month') %}
+            {% set classes_month = classes_month + ' govuk-input--error' %}
+          {% endif %}
+           {% if validationResult.getErrorForField('year') %}
+            {% set classes_year = classes_year + ' govuk-input--error' %}
+          {% endif %}
+          {% if validationResult.getErrorForField('startDate') %}
+            {% set classes_day = classes_day + ' govuk-input--error' %}
+            {% set classes_month = classes_month + ' govuk-input--error' %}
+            {% set classes_year = classes_year + ' govuk-input--error' %}
+          {% endif %}
+
+          {% set errorMessages %}
+            <ul style="padding: 0;list-style-type: none;">
+              {% for error in validationResult.errors %}
+                <li>
+                  {{ error.text }}
+                </li>
+              {% endfor %}
+            </ul>
+          {% endset %}
+
+          {% if errorMessages %}
```

Comment:
That 'if' statement is not clear to me. When I read it I think it is always set due to `set` clause above. If so there is no point having 'if' statement.

Reply:
good point, i've removed the if statement now